CHANGELOG.md

```diff
@@ -2,7 +2,10 @@
 
 ## unreleased
 
```

Comment:
I don't see an entry for delayed-compaction https://github.com/prometheus/prometheus/pull/12532#issuecomment-2407032854